### DISCUSSION TITLE
【求助】我穿越成反派干部了

### DISCUSSION THREAD

**Discussion by tiancaitu** [share121/inter-knot#1439] [其他]
穿越成这个世界的反派干部了，名字好像跟小丑有关?
这里主角刚刚收集完五元素之力?而且这个世界的主角是黄色的?黄毛主角，元素力量，反派干部?
我穿越到原神了?
跟小丑有关?我不会成愚人众第一席的丑角了吧?
原来是这个主角啊?这fw黄毛搜集完了五元素却连仆人都打不过，我这不是轻松拿捏?
等会!你怎么是黑发的?黄毛呢?
什么叫“你将代表世间的正义力量”?
他长这样，会赢吗？
![20230109204728_f6af0](https://github.com/user-attachments/assets/4bac0f05-dc28-4c91-905b-5d1fefef1b80)

**Comment by kimile30** [comment]
是四元素合一成的雷😶

**Comment by piaomiaoxing** [comment]
宁...自求多福吧，记住别求饶，他不会听的
